infra/nginx.conf

```diff
@@ -1,6 +1,7 @@
 server_tokens off;
 
 server {
+        listen 80;
         listen 443 ssl;
 
         server_name www.grtp.co;
```

Comment:
You need to add `grtp.co` in the list, no? To handle `<script src="http://grtp.co">…` correctly.